errbot-reminder.py

```diff
@@ -0,0 +1,74 @@
+from errbot import BotPlugin, botcmd
+from datetime import datetime, time, timedelta, date
+
+# Daily / Retrospective --> If it's an ordinary day or a retrospective, meeting at 9:30
+# Sprint Planning --> If it's a Sprint planning, meeting at 15:30
+# Sprint review --> If it's a Sprint Review, meeting at 14:45
+
+class PluginTemplate(BotPlugin):
+  @botcmd
+  def whereami(self, msg, args):
```

Comment:
This is the name of the command you will run in the chat. 
Unfortunately you can't use a command for this. We want this to run automatically at regular intervals.